### PR TITLE
Points Multiplier - Shop [Beta]

### PR DESCRIPTION
**Pull request by thexpiredpear**
Added a points multiplier (perm) in the shop

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -380,13 +380,17 @@ async def truefalse(ctx, category=None):
         multiplier = 1.5
     else:
         multiplier = 1
+    if tbperms("check", ctx.message.author.id, "1.5x"):
+        mult2 = 1.5
+    else: mult2 = 1.5
```
Comment (gubareve) [inline bot.py:385]: it should be 1 without buying

File: bot.py [hunk 2]
```diff
@@ -380,13 +380,17 @@ async def truefalse(ctx, category=None):
         multiplier = 1.5
     else:
         multiplier = 1
+    if tbperms("check", ctx.message.author.id, "1.5x"):
```
Comment (gubareve) [inline bot.py:383]: author id should be a string

File: bot.py [hunk 3]
```diff
@@ -546,7 +550,14 @@ async def multichoice(ctx, category=None):
             diduservote = False
         pointstogive = 1 if category in categories.keys() else 2
         if diduservote:
-            pointstogive += 1.5
+            mult = 1.5
+        else:
+            mult = 1
+        if tbperms("check", ctx.message.author.id, "1.5x"):
```
Comment (gubareve) [inline bot.py:556]: author id should be a string

File: bot.py (hunk 1 above)
Comment (thexpiredpear) [reply bot.py:385]: crap bug

**Comment by gubareve** [comment]
@all-contributors add @thexpiredpear for code, design

**Pull request merged**